### ISSUE TITLE
Incomplete dts for haydn and venus

### ISSUE THREAD

**Issue by atomsand** [MiCode/kernel_devicetree#9]
`qcom/camera`and`qcom/display`is missing

**Comment by mrslezak** [comment]
Yes this won't build without it!!! Please upload!  I tried the camera from the S21 Ultra but it's different even though the same display panel.  #include "camera/haydn-sm8350-camera-sensor.dtsi" does not exist in other repos - it's Xiaomi specific.

**Comment by mrslezak** [comment]
Found that they just posted these in the star-r-oss device tree - go clone from there and just put camera and display into your device tree.
